CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# How to Contribute
+
+* Reach out to us on our public [Slack channel](https://www.wavefront.com/join-public-slack).
+* If you run into any issues, let us know by creating a GitHub issue.
+
+## Testing
+
+We have a handful of GitHub actions to check code and run tests, but most of our testbed consists of Python [unittest](https://docs.python.org/3/library/unittest.html). Please write Python [unittest](https://docs.python.org/3/library/unittest.html#basic-example) for new code you create.
```

Comment:
..., but most of our testbeds consist of Python [unittest](https://docs.python.org/3/library/unittest.html). Please write Python [unittest](https://docs.python.org/3/library/unittest.html#basic-example) for the new code that you create.